doc/api/timers.md

```diff
@@ -316,7 +317,8 @@ Cancels a `Timeout` object created by [`setInterval()`][].
 added: v0.0.1
 -->
 
-* `timeout` {Timeout} A `Timeout` object as returned by [`setTimeout()`][].
+* `timeout` {Timeout|string|number} A `Timeout` object as returned by [`setTimeout()`][]
+  or the ID of the `Timeout` object as a string or a number.
```

Comment:
Link to `#timers_timeout_symbol_toprimitive` so people know how to get the ID?

Reply:
Also, we don't call it an _ID_ in  `#timers_timeout_symbol_toprimitive` so maybe either alter the text in this PR or in that section so that the terminology is consistent?